src/components/ChannelPreviewLastMessage.js

```diff
@@ -37,7 +46,7 @@ export class ChannelPreviewLastMessage extends PureComponent {
 
   render() {
     const unreadClass =
-      this.props.unread_count >= 1 ? 'str-chat__channel-preview--unread' : '';
```

Comment:
@jaapbakker88 please take a look at this. I tested this and it works fine now, but need second pair eyes!